src/js/components/sharedComponents/StayInTouch.jsx

```diff
@@ -55,12 +66,16 @@ const StayInTouch = (pageName) => {
                                 Sign up to receive our release notes to keep up with what’s new on USAspending.gov.
                         </div>
                         <div className="stay-in-touch__second-row-link">
-                            <a
-                                href="mailto:join-usaspending@lists.fiscal.treasury.gov?subject=Yes!%20I'd%20like%20to%20receive%20updates."
-                                onClick={trackLinkSignUp}>
-                                    Sign Up
-                                <FontAwesomeIcon icon={faArrowRight} />
-                            </a>
+                            <Button
+                                copy="Sign Up"
+                                buttonTitle="Sign Up"
+                                buttonSize="md"
+                                onClick={handleSignUp}
+                                buttonType="text"
+                                backgroundColor="light"
+                                imageAlignment="right"
+                                textAlignment="left"
+                                image={<FontAwesomeIcon icon={faArrowRight} />} />
```

Comment:
The font size here is much larger than PROD.  Could you try overriding the icon size here so it matches what's in PROD?

Reply:
Just to make sure, Marcy did say it would be fine if they are 24px and  text aligned center.
I have no problem overriding though